### ISSUE TITLE
Missing historical data in streaming feather

### ISSUE THREAD

**Issue by rsmb7z** [nautechsystems/nautilus_trader#1042]
# Feature Request

Not sure if it is skipped intentionally, when the historical response is returned using `_handle_bars` or `_handle_quote_ticks` by the adapter it is not recorded in the streaming feather. However if returned through `_handle_data` it does record.

**Comment by cjdsellers** [comment]
So for streaming data, I see we always publish on the bus like this:
```
self._msgbus.publish_c(topic=f"data.bars.{bar_type}", msg=bar)
```

We don't do this for responses though currently. The thinking there is that a response is specific to the request, and so its more of a point-to-point messaging pattern than Pub/Sub where many consumers may be interested in a continuous data stream. 

If you could share your use case a little more, were you just wanting to capture all data events including the responses and persist them to the catalog?

**Comment by rsmb7z** [comment]
> If you could share your use case a little more, were you just wanting to capture all data events including the responses and persist them to the catalog?

Yes correct, to capture all into catalog. Basically I am trying to use Actor for historical data and using request from DataEngine and was expecting the request data will log in catalog.
